lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -20,19 +26,29 @@ additional_paths_install:
 additional_paths_receive:
   kind: boolean
   config_get_token_append: '/^additional-paths receive$/'
-  config_set_append: '<state> additional-paths receive'
+  cli_nexus:
+    config_set_append: '<state> additional-paths receive'
+  cli_ios_xr:
+    config_set_append: ' <state> additional-paths receive'
   default_value: false
 
 additional_paths_selection:
   kind: string
-  config_get_token_append: '/^additional-paths selection route-map (.*)$/'
-  config_set_append: '<state> additional-paths selection route-map <route_map>'
+  cli_nexus:
+    config_get_token_append: '/^additional-paths selection route-map (.*)$/'
+    config_set_append: '<state> additional-paths selection route-map <route_map>'
+  cli_ios_xr:
+    config_get_token_append: '/^additional-paths selection route-policy (.*)$/'
+    config_set_append: ' <state> additional-paths selection route-policy <route_policy>'
```

Comment:
Since Nexus never has a route-policy and XR never has a route-map, and they are used for similar purposes, would it make more sense to have both OS share a single <route_map_policy> key?